doc/cmake4vim.txt

```diff
@@ -63,6 +63,13 @@ COMMANDS                                                        *cmake-commands*
     Run the current target if it is runnable and compiled.
     Command allows to reset arguments from vimspector config
 
+:CCMake!                                                          *:CCMake!*
```

Comment:
`:CCMake`, there is no `-bang` attribute for the function.

Reply:
Copy-past issue. Thanks!